konfera/models/schedule.py

```diff
@@ -23,3 +25,35 @@ class Schedule(KonferaModel):
 
     def __str__(self):
         return _('%(start)s (%(duration)s min)') % {'start': self.start, 'duration': self.duration}
+
+    def clean(self, *args, **kwargs):
+        # Only approved talk can be scheduled
+        if self.talk.status != 'approved':
```

Comment:
@ZoltanOnody i read from the other Issue that the constant will be transfered from global to models (inside). so maybe i'll just wait for it to be merged then do the changes :)